Userland/Libraries/LibWeb/Layout/TreeBuilder.cpp

```diff
@@ -380,11 +380,11 @@ ErrorOr<void> TreeBuilder::create_layout_tree(DOM::Node& dom_node, TreeBuilder::
         auto& parent = *dom_node.layout_node();
 
         // If the box does not overflow in the vertical axis, then it is centered vertically.
-        auto table_computed_values = CSS::ComputedValues();
+        auto table_computed_values = parent.computed_values().clone_inherited_values();
         static_cast<CSS::MutableComputedValues&>(table_computed_values).set_display(CSS::Display::from_short(CSS::Display::Short::Table));
         static_cast<CSS::MutableComputedValues&>(table_computed_values).set_height(CSS::Size::make_percentage(CSS::Percentage(100)));
 
-        auto cell_computed_values = CSS::ComputedValues();
+        auto cell_computed_values = parent.computed_values().clone_inherited_values();
         static_cast<CSS::MutableComputedValues&>(cell_computed_values).set_display(CSS::Display { CSS::Display::Internal::TableCell });
         static_cast<CSS::MutableComputedValues&>(cell_computed_values).set_vertical_align(CSS::VerticalAlign::Middle);
         static_cast<CSS::MutableComputedValues&>(cell_computed_values).set_white_space(CSS::WhiteSpace::Nowrap);
```

Comment:
Can we remove this now, since white-space is inherited?

Reply:
Done.